pkg/infrastructure/azure/azure.go

```diff
@@ -784,6 +809,22 @@ func (p Provider) Ignition(ctx context.Context, in clusterapi.IgnitionInput) ([]
 			return nil, fmt.Errorf("failed to create PageBlob for ignition shim: %w", err)
 		}
 	}
+
+	// Should we separate azurestack from azure when using the user delegated SAS?
+	// Azurestack is generating an SAS url at this point.
```

Comment:
This is a really good question. I have checked the azure-sdk-for-go and I don't see any azure stack support for these packages in the go sdk, so I think we will have to stick with shared key access on azure stack